protocol/src/main/java/org/threadly/litesockets/protocols/http/request/HTTPRequestBuilder.java

```diff
@@ -178,28 +178,48 @@ public HTTPRequestBuilder removeQuery(final String key) {
    * Sets the {@link HTTPAddress} for this builder.  This will add a Host header into the headers of this builder
    * when this object it built.  This is also used with the {@link #buildHTTPAddress()} method.
    * 
+   * NOTE: This does not change the Host HTTP header
```

Comment:
I just noticed this, why?  I thought most http clients did this by the default.

Reply:
Maybe this should just accept a boolean so that the user is forced to consider what behavior they want?  I agree it should be possible (easy even), but also feel like it's a rare use case, and people are more likely to be bitten by it not being set vs it being set too much.

Reply:
I did that for the setHost method.  I went ahead and added it for this one as well.